packages/global/src/global.ts

```diff
@@ -36,6 +36,7 @@ if (typeof __DEV__ !== "boolean") {
     )
   ) {
     global["__DEV__"] = true;
+    console.warn('SET DEV TO TRUE')
```

Comment:
Maybe helps make it more descriptive, in case we see it happening outside of Jest environments

```suggestion
    console.warn('SET DEV TO TRUE FOR JEST');
```

Reply:
I noticed that makes test outputs very noisy. I was planning to remove it for the final code.